pkg/envinfo/envinfo.go

```diff
@@ -135,6 +138,11 @@ func newProxyEnvInfo() proxyEnvInfo {
 func (esi *EnvSpecificInfo) SetConfiguration(parameter string, value interface{}) (err error) {
 	if parameter, ok := asLocallySupportedParameter(parameter); ok {
 		switch parameter {
+		case "create":
+			createValue := value.(ComponentSettings)
+			esi.componentSettings.Type = createValue.Type
+			esi.componentSettings.Name = createValue.Name
```

Comment:
we should have a GetName() func like GetURL() to read the component name for usage in devfile.go(see my other review comment)

Reply:
@maysunfaisal Answer in your other review, should handle the two comments together